packages/schematics/src/collection/ngrx/schema.json

```diff
@@ -30,6 +30,12 @@
       "description":
         "Add StoreModule.forRoot and EffectsModule.forRoot() instead of forFeature (e.g., --root)."
     },
+    "facade": {
+      "type": "boolean",
+      "default": false,
+      "description":
+        "Create a Facade class for the the Feature (e.g., --facade)."
```

Comment:
Can you create facade together with root state?

Reply:
Sure, why not? Or are you seeing something I am missing?